src/main/kotlin/com/atlassian/performance/tools/report/api/judge/RelativeNonparametricPerformanceJudge.kt

```diff
@@ -2,7 +2,7 @@ package com.atlassian.performance.tools.report.api.judge
 
 import com.atlassian.performance.tools.jiraactions.api.ActionType
 import com.atlassian.performance.tools.report.ActionMetricsReader
-import com.atlassian.performance.tools.report.api.ShiftedDistributionRegressionTest
+import com.atlassian.performance.tools.report.api.distribution.DistributionComparator
```

Comment:
No difference detected by `RelativeNonparametricPerformanceJudgeTest`?

Reply:
No diff here

Reply:
So we don't see why the new one is better.

Reply:
How do you define better? Classification has not changed for unit tested cases and as stated in first PR comment, it's expected